frontend/src/pages/Dashboard/index.tsx

```diff
@@ -150,473 +60,460 @@ const columns = [
-    modelUser: 'Yuanming Lee',
-    modelDataset: 'GLUE',
-    hasDetail: false,
-    servingDevice: 'Nvidia Tesla P4',
-    allThroughput: '270.341 req/sec',
-    allLatency: '26.580 sec',
-    batchSize: 64,
-    batchNum: 100,
-    totalMem: '7981694976.0 bytes',
-    usedMem: '7763132416.0 bytes',
-    memPer: '0.972',
-    gpuUtil: '75.653%',
-    latency50: '0.265 s',
-    latency95: '0.276 s',
-    latency99: '0.279 s',
-  },
-  {
-    key: 1,
-    modelId: '734d623fd24',
-    modelName: 'ResNet-50',
-    modelFramework: 'PyTorch',
-    convertVersion: 'TensorFlow-SavedModel',
-    modelAcc: '70.1%',
-    modelEngine: 'TorchScript',
-    modelTask: 'image classification',
-    modelUser: 'Yizheng Huang',
-    modelDataset: 'ImageNet',
-    hasDetail: false,
-    servingDevice: 'Nvidia Tesla P4',
-    allThroughput: '200.333 req/sec',
-    allLatency: '31.946 sec',
-    batchSize: 16,
-    batchNum: 100,
-    totalMem: '7981694976.0 bytes',
-    usedMem: '7763132416.0 bytes',
-    memPer: '97.263%',
-    gpuUtil: '69.871%',
-    latency50: '78.906 ms',
-    latency95: '84.170 ms',
-    latency99: '86.797 ms',
-  },
-];
+const getRandomuserParams = (params) => {
+  return {
+    results: params.pagination.pageSize,
+    page: params.pagination.current,
+    ...params,
+  };
+};
 
-const Dashboard = () => {
-  return (
-    <Card>
-      <div
-        style={{ marginBottom: '20px', display: 'flex', flexDirection: 'row' }}
-      >
-        <Button
-          size="large"
-          type="primary"
-          onClick={() => {
-            console.log('You clicked Register Model');
-          }}
-        >
-          Register Model
-        </Button>
-        <Button
-          size="large"
-          style={{ marginLeft: '5px' }}
-          onClick={() => {
-            console.log('You clicked Download Table');
+export default class Dashboard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      profilingResult: {},
+      allModelInfo: [],
+      pagination: {
+        current: 1,
+        pageSize: 10,
+      },
+      loading: false,
+    };
+    this.loadAllModels();
+  }
+
+  componentDidMount() {
+    const { pagination } = this.state;
+    this.fetch({ pagination });
+  }
+
+  handleTableChange = (pagination, filters, sorter) => {
+    this.fetch({
+      sortField: sorter.field,
+      sortOrder: sorter.order,
+      pagination,
+      ...filters,
+    });
+  };
+
+  fetch = (params = {}) => {
+    this.setState({ loading: true });
+    reqwest({
+      url: '<your API address>',
+      method: 'get',
+      type: 'json',
+      data: getRandomuserParams(params),
+    }).then((allModelInfo) => {
+      this.setState({
+        loading: false,
+        data: allModelInfo.data,
+        pagination: {
+          ...params.pagination,
+          total: this.state.allModelInfo.length,
+        },
+      });
+    });
+  };
+
+  loadAllModels = () => {
+    const targetUrl = '<your API address>';
```

Comment:
How do users set up this IP? Automatically?

Reply:
Do we have a clear instruction or an automatic bash script for this?

Reply:
user needs to change the code here. It's impossible for us to know where they deploy modelci. I'm quite confused about why you have issues here.

Reply:
> Do we have a clear instruction or an automatic bash script for this?

no need

Reply:
frontend and backend are separated

Reply:
Actually. I suppose it is simple that a user can deploy a project with Web UI with no pain as shown below.

![image](https://user-images.githubusercontent.com/5894780/86007695-57b44680-ba4a-11ea-9fb7-b7660b319c5d.png)

Reply:
no need here.

Reply:
for modelci, our user can deploy in cloud or locally. the IP address should be filled by themselves.

Reply:
https://github.com/mlflow/mlflow

I think we can get convergence.

Like promethues and all similar projects, we just need to hardcode 127.0.0.1 here. Once users start our project, they can go to the website.

Reply:
if only for local usage, we can hardcode http://localhost:5000 or something here, but we should allow cloud serving since we need profiling in could GPUs. The best and simplest way is left here to fill.

Reply:
I'll update the doc.

Reply:
OK. Two things we can get convergence.

1. hardcoding a localhost so that all users and developers can test our project on their own with no pain.

2. A simple instruction for cloud users to follow so that they know how to fill up their IP address in your source code of Web UI

Reply:
sure